### PR TITLE
Feat: Confirm email page

### PR DESCRIPTION
**Pull request by mariusjagminas**

### CODE REVIEW

File: packages/webapp/src/pages/ConfirmEmail/ConfirmEmail.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        setIsLoading(false)
+        setShouldShowError(true)
+      }
+    }
+
+    void verifyEmail()
+  }, [params])
+
+  if (isLoading) {
+    // Add verififying and error pages when desings are ready.
```
Comment (kowalski) [inline packages/webapp/src/pages/ConfirmEmail/ConfirmEmail.tsx:35]: please remember to ask Darek for it

Comment (kowalski) [reply packages/webapp/src/pages/ConfirmEmail/ConfirmEmail.tsx:35]: or better - note it down as issue

File: packages/webapp/src/pages/TurntableRoom.tsx [hunk 2]
```diff
@@ -22,6 +22,10 @@ export const TurntableRoom: React.FC = (): JSX.Element => {
   const [avatarId, setAvatarId] = useState('')
   const [userName, setUserName] = useState('')
 
+  useEffect(() => {
+    dispatch(sessionActions.validate())
```
Comment (kowalski) [inline packages/webapp/src/pages/TurntableRoom.tsx:26]: I'm not sure why it should be responsibility of the Page to trigger this.
Imo it should auto-trigger from session saga

File: packages/webapp/src/pages/ConfirmEmail/ConfirmEmail.tsx (hunk 1 above)
Comment (mariusjagminas) [reply packages/webapp/src/pages/ConfirmEmail/ConfirmEmail.tsx:35]: ok i'll create an issue.

**Review by kowalski — APPROVED** [review]

**Pull request merged**